openshift/tests-extension/README.md

```diff
@@ -158,6 +158,14 @@ This ensures compatibility when running tests in non-OpenShift environments such
     - Run `make update-metadata` or `make build-update`
     - Run `make verify` to check formatting, linting, and validation
 
+**IMPORTANT** Ensure that you either test any new test with `/payload-aggregate`
+to avoid issues with Sippy or other tools due flakes.
+
+**Examples**
+
+- For `[Serial]` tests run: `/payload-aggregate periodic-ci-openshift-release-master-ci-4.20-e2e-gcp-ovn-techpreview-serial 10`
```

Comment:
What do the 10 and 5 mean? Is there any canonical list of test environments documented somewhere?

Reply:
Maybe we can add in the desc ( at least 5 times ?! at minimal )